### ISSUE TITLE
Split Screen/Multi-Window not working

### ISSUE THREAD

**Issue by HurricAngel** [markusfisch/PieLauncher#146]
Whenever I try to use the split screen function an error message will pop up saying "Cannot use this function when Pie Launcher is in use." I can access multi-window if I go to another launcher like Xperia Home or Nova launcher, and force stop Pie launcher. It is even possible to switch back to the previous multi-window setup if I had it activate first, then launches Pie launcher back up. But if I try to activate a different pair of multi-window the same error message pops up again.

I am using a Sony Xperia 1 VI on Android 15, not sure if it is a Sony issue or a common issue.

Sidenote: the pop up window function right next to multi-window works perfectly fine with Pie Launcher.

![Screenshot_20241215-204424](https://github.com/user-attachments/assets/f1a60fa4-f0fe-4e66-88ac-de2a0f5115d6)

**Comment by markusfisch** [comment]
Thanks for filling an issue about this! 👍 I had no idea this isn't working. Will have to investigate.

**Comment by HurricAngel** [comment]
I was messing with the settings today and accidentally found a workaround for this issue.

All you have to do is turn on "Enable non-resizable in multi window" in dev options and multi window now works with Pie Launcher. So it is safe to assume Pie Launcher is classified as "non-resizable" in some way?
![Screenshot_20241219-115816](https://github.com/user-attachments/assets/fac4fd4e-4295-46c0-9b61-a2e1df2b1b71)

**Comment by markusfisch** [comment]
Yes, exactly! I didn't set the app as [resizeable](https://developer.android.com/guide/topics/manifest/application-element#resizeableActivity) because I didn't think that might be relevant for a launcher 🙈 Sorry, I could have thought of that myself.

Fixed in https://github.com/markusfisch/PieLauncher/commit/e14905aca63a22b670f913dbeeb75db0afc36166

Will be in the next version!

**Comment by HurricAngel** [comment]
On my phone, you need Developer option AND "Enable non-resizable in multi window" enabled for it to work.

The latest release on Google Play fixed this I think.
